include/libtorrent/aux_/ffs.hpp

```diff
@@ -34,7 +34,6 @@ POSSIBILITY OF SUCH DAMAGE.
 #define TORRENT_FFS_HPP_INCLUDE
 
 #include <cstdint>
-#include "libtorrent/export.hpp"
```

Comment:
sure, there are transitive dependencies on that header elsewhere, that doesn't make it a good idea to depend on them. This is what include guards are for.